.github/workflows/cancel.yml

```diff
@@ -0,0 +1,15 @@
+name: Cancel
```

Comment:
This workflow will cancel duplicate CI builds. For example, when rapidly pushing commits to a PR, if there are CI builds still running from an older commit they will be cancelled